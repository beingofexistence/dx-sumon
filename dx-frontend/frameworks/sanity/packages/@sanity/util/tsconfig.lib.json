{
  "extends": "../../../tsconfig.settings",
  "include": ["./exports", "./src", "./typings"],
  "exclude": [
    "./src/**/__fixtures__",
    "./src/**/__mocks__",
    "./src/**/__workshop__",
    "./src/**/*.test.ts",
    "./src/**/*.test.tsx"
  ],
  "compilerOptions": {
    "composite": true,
    "rootDir": ".",
    "outDir": "./lib"
  },
  "references": [{"path": "../types/tsconfig.lib.json"}]
}
