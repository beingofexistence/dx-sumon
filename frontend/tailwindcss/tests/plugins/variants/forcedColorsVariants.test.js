import { quickVariantPluginTest } from '../../util/run'

quickVariantPluginTest('forcedColorsVariants').toMatchSnapshot()
