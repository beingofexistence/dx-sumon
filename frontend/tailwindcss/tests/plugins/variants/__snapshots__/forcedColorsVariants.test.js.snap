// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test the 'forcedColorsVariants' plugin 1`] = `
"
@media (forced-colors: active) {
  .forced-colors\\:flex {
    display: flex;
  }
}
"
`;
